platform/security/platform-security-core-api/src/main/java/ddf/security/SubjectUtils.java

```diff
@@ -32,11 +33,37 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(SubjectUtils.class);
 
+    public static final String GUEST_DISPLAY_NAME = "Guest";
+
     private SubjectUtils() {
 
     }
 
     /**
+     * Converts the given principal name to a formatted display name.
+     *
+     * @param principal
+     * @param defaultName
+     * @return
+     */
+    private static String getDisplayName(Principal principal, String defaultName) {
+
+        String displayName = defaultName;
+
+        if (principal instanceof GuestPrincipal) {
+            displayName = GUEST_DISPLAY_NAME;
+        } else if (principal instanceof X500Principal) {
+            getCommonName((X500Principal) principal);
+        } else {
```

Comment:
Shouldn't the else case be principal.getName()? Check that there is a default before returning it?